release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -3721,3 +3721,35 @@ After this one-time manual restart, subsequent upgrades and rotations will
 ensure restart before the service CA expires without requiring manual
 intervention.
 ====
+
+[[ocp-4-4-27]]
+=== RHBA-2020:4063 - {product-title} 4.4.27 Bug Fix Update
+
+Issued: 2020-10-13
+
+{product-title} release 4.4.27 is now available. The list of container images and bug fixes included in the update are documented in the link:https://access.redhat.com/errata/RHBA-2020:4063[RHBA-2020:4063] advisory. The RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:4064[RHBA-2020:4064] advisory.
```

Comment:
Just want to sanity check that these links are 404 because the content is not yet live, correct?